### ISSUE TITLE
REPL does not start with IntelliJ IDEA 2022.2.1

### ISSUE THREAD

**Issue by mouse07410** [rikvdkleij/intellij-haskell#691]
macOS Monterey 12.5.1, IntelliJ IDEA 2022.2.1

GHC-9.0.2 (and GHC-9.2.4), Cabal-3.8.1.0, Stack-2.7.5. IntelliJ-Haskell 1.0.0beta88.

I keep getting error messages
```
[project-stack-repl-hask-tst3:lib] REPL couldn't be started within 30 seconds
```

Also, not sure if this plugin is supposed to be able to create a new Haskell project - but it fails upon attempt to do so.

**Comment by Him188** [comment]
Seems I have the same problem. I rolled back to 2022.1 and it works.

**Comment by mouse07410** [comment]
Given that this plugin doesn't work with the current IntelliJ IDEA and with the current GHC - instead of downgrading IDEA, I dumped this plugin and for Haskell switched to Visual Studio Code with Haskell Language Server and appropriate  working VSC plugins. 

While there are some problems with Haskell debugger under VSC - that setup is far superior to what this plugin could ever offer. Not to mention that under VSC one can build projects with Cabal and Stack. 

In short, I recommend switching to VSC for Haskell.
